src/main/java/fi/otavanopisto/kuntaapi/server/integrations/ptv/updaters/PtvServicePriorityIdUpdater.java

```diff
@@ -19,6 +19,8 @@
 @SuppressWarnings ("squid:S3306")
 public class PtvServicePriorityIdUpdater extends AbstractPtvServiceIdUpdater {
   
+  private final static int UPDATE_SLACK_MINUTE = 3;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)